src/RestApi/StoreApi/Schemas/CartItemSchema.php

```diff
@@ -83,31 +83,43 @@ protected function get_properties() {
 				'items'       => [
```

Comment:
I'm still seeing currency related properties in the response which aren't in the schema, e.g. `item.totals.currency_code` (API response from this branch):

<img width="657" alt="Screen Shot 2020-01-14 at 3 21 38 PM" src="https://user-images.githubusercontent.com/4167300/72308620-b460b680-36e1-11ea-828c-0cff0322dd1a.png">

Reply:
I thin you're looking at the schema wrong :) See there is an array merge of `get_store_currency_properties` - this merges the currency schema in.

Confirm this by doing an OPTIONS request to `wc/store/cart`. Those properties are included. Also unit tests would fail if mismatched.

Reply:
Ah I see, thanks for explaining :)

Reply:
Confirmed with `curl -X OPTIONS`, I see the currency fields 🎉